src/views/SelectProduct.vue

```diff
@@ -157,14 +157,25 @@ export default defineComponent({
 </script>
 
 <style scoped>
-.filters {
-  border-right: 1px solid var(--ion-color-medium);
-}
 .section-grid {
   grid-template-columns: repeat(auto-fill, 200px);
 }
 
 @media (min-width: 991px) {
+  .find {
+    grid: "search  filters"
+          "main    main"
+          / 440px 400px;        
+    justify-content: space-between;   
+    row-gap: var(--spacer-xl);
+    margin: var(--spacer-lg) var(--spacer-base);               
+  }
+
+  main {
+    padding: var(--spacer-sm) 0;
+    border-top: 1px solid var(--ion-color-medium)
```

Comment:
we can introduce global border variable here

Reply:
Ok Sir, I have replaced this value with border varibale.